File: HB/pysrc/프로그래머스/레벨0/Day16/문자열_계산하기.py
def solution(my_string):
    answer = 0
    a= my_string.split(' ')
    while len(a)!=1:
        if a[1]=='+':
            answer= int(a[0])+int(a[2])
            a.pop(0)
            a.pop(0)
            a.pop(0)
            a.insert(0,answer)
        else:
            answer= int(a[0])-int(a[2])
            a.pop(0)
            a.pop(0)
            a.pop(0)
            a.insert(0,answer)
    return answer

File: SB/src/프로그래머스/레벨0/Day21/문자열_계산하기.java
package 프로그래머스.레벨0.Day21;

public class 문자열_계산하기 {
    class Solution {
        public int solution(String my_string) {
            String sArr[] = my_string.split(" ");
            int answer = Integer.parseInt(sArr[0]);

            for (int i = 1; i < sArr.length; i++) {
                if (sArr[i].equals("+")) {
                    answer += Integer.parseInt(sArr[i + 1]);
                } else if (sArr[i].equals("-")) {
                    answer -= Integer.parseInt(sArr[i + 1]);
                }
            }

            return answer;
        }
    }
}
